webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -97,8 +183,78 @@ function GithubLabels(props) {
     }) : null;
 }
 
+function getReviewText(item, style, secondLine) {
+    let reviews = '';
+    let changes = '';
+
+    const finishedReviewers = [];
+
+    const reverse = (accum, cur) => {
+        accum.unshift(cur);
+        return accum;
+    };
+
+    const lastReviews = item.reviews.reduce(reverse, []).filter((v) => {
+        if (v.user.login === item.user.login) {
+            return false;
+        }
+
+        if (item.requestedReviewers.includes(v.user.login)) {
+            return false;
+        }
+
+        if (v.state === 'COMMENTED' || v.state === 'DISMISSED') {
+            return false;
+        }
+
+        if (finishedReviewers.includes(v.user.login)) {
+            return false;
+        }
+
+        finishedReviewers.push(v.user.login);
+        return true;
+    });
+
+    const approved = lastReviews.reduce((accum, cur) => {
+        if (cur.state === 'APPROVED') {
+            return accum + 1;
+        }
+        return accum;
+    }, 0);
+
+    const changesRequested = lastReviews.reduce((accum, cur) => {
+        if (cur.state === 'CHANGES_REQUESTED') {
+            return accum + 1;
+        }
+        return accum;
+    }, 0);
+
+    const totalReviewers = finishedReviewers.length + item.requestedReviewers.length;
+    if (totalReviewers > 0) {
+        let reviewName;
+        if (totalReviewers === 1) {
+            reviewName = 'review';
+        } else {
+            reviewName = 'reviews';
+        }
+        reviews = (<span>{approved} out of {totalReviewers} {reviewName} complete.</span>);
+    }
+
+    if (changesRequested > 0) {
+        changes = (<div style={{...style.icon, fill: '#c11b28'}}><DotIcon/></div>);
```

Comment:
Could we add a tooltip on this icon to indicate what it means? "Changes Requested"

Reply:
Done!

Reply:
Is it possible to use the requested changes icon from GitHub? 
![image](https://user-images.githubusercontent.com/6913320/73301362-a1b9a780-41e0-11ea-9771-7996b6d62814.png)

Reply:
I can copy/paste the svg from mattermost site, if that is OK (similar to what I did with the tick, cross and dot icons)

Reply:
Do you mean GitHub? Getting it straight from there sounds fine to me.

Reply:
Yeah, sorry. I meant GitHub. Tomorrow I will get that done first thing in the morning :D

Reply:
Done
![image](https://user-images.githubusercontent.com/1933730/73341069-2e657380-427c-11ea-9778-4b1754f8f456.png)
I feel the icon is still a bit thin, but apart of that I like the result.

Reply:
The requested changes icon seems like it should be more "in your face", as it represents an action item you should complete. But placing it next to the number of reviews seems to make sense.

@marianunez What do you think about putting it next to the build status icon instead, so it is more visible? I don't want to keep dragging this PR along, but I also want to get it right as we're making this final push.

Also, if the PR's name is much longer, I wonder how the RHS display looks in general.

Reply:
0/5 I think I prefer on the bottom, two icons next to each other may seem confusing. 
However, deferring to @asaadmahmood expertise on this.

Reply:
@mickmister Here I leave you a screenshot with real data. Sadly, I have no PR with changes requested at the moment, so the icon will not appear.

![image](https://user-images.githubusercontent.com/1933730/73380142-cdfa2480-42c3-11ea-83c0-d54ef3723c30.png)